src/editmap.cpp

```diff
@@ -573,119 +506,136 @@ void editmap::update_view( bool update_info )
+        mvwputch( w_info, point( 2, off ), furniture_type.color(), furniture_type.symbol() );
+        mvwprintw( w_info, point( 4, off ), _( "%d: %s; movecost %d movestr %d" ),
+                   g->m.furn( target ).to_i(),
+                   furniture_type.name(),
+                   furniture_type.movecost,
+                   furniture_type.move_str_req
                  );
-        off++; // 2
-        if( g->m.furn( target ) > 0 ) {
-            mvwputch( w_info, point( 2, off ), furniture_type.color(), furniture_type.symbol() );
-            mvwprintw( w_info, point( 4, off ), _( "%d: %s; movecost %d movestr %d" ),
-                       g->m.furn( target ).to_i(),
-                       furniture_type.name(),
-                       furniture_type.movecost,
-                       furniture_type.move_str_req
-                     );
-            off++; // 3
-        }
-        const auto &map_cache = g->m.get_cache( target.z );
-
-        mvwprintw( w_info, point( 1, off++ ), _( "dist: %d u_see: %d v_in: %d scent: %d" ),
-                   rl_dist( g->u.pos(), target ), static_cast<int>( g->u.sees( target ) ),
-                   veh_in, g->scent.get( target ) );
-        mvwprintw( w_info, point( 1, off++ ), _( "sight_range: %d, daylight_sight_range: %d," ),
-                   g->u.sight_range( g->light_level( g->u.posz() ) ),
-                   g->u.sight_range( current_daylight_level( calendar::turn ) ) );
-        mvwprintw( w_info, point( 1, off++ ), _( "transparency: %.5f, visibility: %.5f," ),
-                   map_cache.transparency_cache[target.x][target.y],
-                   map_cache.seen_cache[target.x][target.y] );
-        map::apparent_light_info al = map::apparent_light_helper( map_cache, target );
-        int apparent_light = static_cast<int>(
-                                 g->m.apparent_light_at( target, g->m.get_visibility_variables_cache() ) );
-        mvwprintw( w_info, point( 1, off++ ), _( "outside: %d obstructed: %d" ),
-                   static_cast<int>( g->m.is_outside( target ) ),
-                   static_cast<int>( al.obstructed ) );
-        mvwprintw( w_info, point( 1, off++ ), _( "light_at: %s" ),
-                   map_cache.lm[target.x][target.y].to_string() );
-        mvwprintw( w_info, point( 1, off++ ), _( "apparent light: %.5f (%d)" ),
-                   al.apparent_light, apparent_light );
-        std::string extras;
-        if( veh_in >= 0 ) {
-            extras += _( " [vehicle]" );
-        }
-        if( g->m.has_flag( TFLAG_INDOORS, target ) ) {
-            extras += _( " [indoors]" );
-        }
-        if( g->m.has_flag( TFLAG_SUPPORTS_ROOF, target ) ) {
-            extras += _( " [roof]" );
-        }
+        off++; // 3
+    }
+    const auto &map_cache = g->m.get_cache( target.z );
+
+    mvwprintw( w_info, point( 1, off++ ), _( "dist: %d u_see: %d v_in: %d scent: %d" ),
+               rl_dist( g->u.pos(), target ), static_cast<int>( g->u.sees( target ) ),
```

Comment:
```suggestion
               rl_dist( g->u.pos(), target ), g->u.sees( target ) ? _("visible"): "_("invisible"),
```
Most people without a programming background won't get what the 1 or 0 there means.